tests/test_integrated_channels/test_integrated_channel/test_exporters/test_learner_data.py

```diff
@@ -89,6 +89,12 @@ def setUp(self):
         assert isinstance(self.exporter, LearnerExporter)
         super().setUp()
 
+    def test_default_channel_settings(self):
+        """
+        If you add any settings to the ChannelSettingsMixin, add a test here for the common default value
+        """
+        assert LearnerExporter('fake-user', self.config).INCLUDE_GRADE_FOR_COMPLETION_AUDIT_CHECK is True
```

Comment:
this is what tests that the instance level variable is False for SAP Exporter

Reply:
nice test!